v4/source/posts.yaml

```diff
@@ -216,6 +216,36 @@
         '403':
           $ref: '#/responses/Forbidden'
 
+  '/posts/{post_id}/set-unread':
+    post:
+      tags:
+        - posts
+      summary: Mark channel as not read.
```

Comment:
What about documenting this as `Mark post as unread`? It's not /really/ just marking the post as unread since it will be marking anything newer than this post in the channel unread, but it makes it a little less weird since you're not providing a channel ID. We would just need to make sure that we are very clear about what's happening in the description below